_data/jobs.yml

```diff
@@ -1,3 +1,13 @@
+- expires: 11/28/2021
+  location: UCAR Unidata, Boulder, CO
+  name: Software Engineer - Science Gateway
+  posted: 2021-11-04
+  url: https://ucar.wd5.myworkdayjobs.com/UCAR_Careers/job/Foothills-Lab-4/Software-Engineer-II---Science-Gateway_REQ-2021-323-1
+- expires: 11/28/2021
```

Comment:
and here as well!

Reply:
Thanks! Yes - I copied the date and just tried to get it done between meetings and overlooked to change the format.